templates/components/_nav_mobile.html.twig

```diff
@@ -52,38 +56,36 @@
                     </ul>
                 {% endif %}
                 <ul>
-                {% if has_role_adherent and is_granted('CREATE_COMMITTEE') %}
-                    <ul>
+                    {% if has_role_adherent and is_granted('CREATE_COMMITTEE') %}
+                        <ul>
+                            <li><ahref="{{path('app_adherent_create_committee')}}">Créer un comité</a></li>
```

Comment:
plase put spaces inside brackets.
=> `{{ path('app_adherent_create_committee') }}`

Reply:
and here too `ahref`

Reply:
👌